examples/stable/mongodb/kanister/mongodb-blueprint.yaml

```diff
@@ -0,0 +1,102 @@
+apiVersion: cr.kanister.io/v1alpha1
+kind: Blueprint
+metadata:
+  name: mongodb-blueprint
+actions:
+  backup:
+    type: StatefulSet
+    outputArtifacts:
+      backupInfo:
+        keyValue:
+          backupIdentifier: "{{ .Phases.backupSnapshots.Output.BackupAllInfo }}"
+    phases:
+    - func: BackupDataAll
+      name: backupSnapshots
+      args:
+        namespace: "{{ .StatefulSet.Namespace }}"
+        container: "mongodb-primary"
+        includePath: /bitnami/mongodb
+        backupArtifactPrefix: "{{ .Profile.Location.Bucket }}/mongodb-backups/{{ .StatefulSet.Name }}/rs_backup"
+  restore:
+    type: StatefulSet
+    phases:
+    # Scale down mongodb replicas
+    - func: ScaleWorkload
+      name: shutdownSecondary
+      args:
+        namespace: "{{ .StatefulSet.Namespace }}"
+        name: "my-release-mongodb-secondary"
+        kind: StatefulSet
+        replicas: 0
+    - func: ScaleWorkload
+      name: shutdownPrimary
+      args:
+        namespace: "{{ .StatefulSet.Namespace }}"
+        name: "my-release-mongodb-primary"
+        kind: StatefulSet
+        replicas: 0
+    - func: ScaleWorkload
```

Comment:
QQ: do we need to scale down the arbiters?

Reply:
Yes. As per my observation, when we modify data in primary and secondary mongo replicas while doing the restore, arbiter can't elect primary replica among them. So we need to recreate the pod